theano/sandbox/cuda/basic_ops.py

```diff
@@ -121,14 +121,7 @@ class GpuFromHost(GpuOp):
 
     check_input = False
 
-    def __eq__(self, other):
-        return type(self) == type(other)
-
-    def __hash__(self):
-        return hash(type(self))
-
-    def __str__(self):
-        return 'GpuFromHost'
+    __props__ = ()
```

Comment:
I didn't know this then. But do I need to have `__props__` for the classes that don't have `__init__` ?